dbt_project.yml

```diff
@@ -65,6 +65,14 @@ on-run-end:
 
 models:
   snowplow_attribution:
+    +file_format: >
```

Comment:
Isn't this going to be redundant now after the latest meeting?

Reply:
Yep